endpoints.json

```diff
@@ -25,5 +25,24 @@
         }
       ]
     }
+  },
+
+  "GET /api/articles/article_id": {
+    "description": "serves an object of a article",
+    "queries": ["article_id"],
+    "exampleResponse": {
+      "articles": [
```

Comment:
when responding with a single object it should not be returned within an array, a key of article (singular) and the value of {article object} would be more appropriate.